.jenkins/actions/run_performance.sh

```diff
@@ -3,20 +3,12 @@ set -e -x
 SCRIPT=`realpath $0`
 SCRIPTPATH=`dirname $SCRIPT`
 ROOT_DIR="$(dirname "$(dirname "$SCRIPTPATH")")"
-TESTDATA_PATH="/project/s1053/fv3core_serialized_test_data/"
+TESTDATA_PATH="/project/s1053/fv3core_serialized_test_data"
+FORTRAN_SERIALIZED_DATA_VERSION=7.2.5
 test -n "${experiment}" || exitError 1001 ${LINENO} "experiment is not defined"
 test -n "${backend}" || exitError 1002 ${LINENO} "backend is not defined"
 
-
-if [ "$experiment" = "c12_6ranks_standard" ]; then
-    data_path="${TESTDATA_PATH}7.0.0/c12_6ranks_standard"
-fi
-if [ "$experiment" = "c96_6ranks_baroclinic" ]; then
-    data_path="${TESTDATA_PATH}7.2.3/c96_6ranks_baroclinic"
-fi
-if [ "$experiment" = "c128_6ranks_baroclinic" ]; then
-    data_path="${TESTDATA_PATH}7.2.3/c128_6ranks_baroclinic"
-fi
+data_path=${TESTDATA_PATH}/${FORTRAN_SERIALIZED_DATA_VERSION}/${experiment}
```

Comment:
can we also apply this change in 
https://github.com/VulcanClimateModeling/fv3core/blob/c6b2081fa600bc4ddc80d5fb144386bebfecb71e/.jenkins/actions/full_cache_build.sh#L11-L19

Reply:
Thanks, @twicki, done!